README.md

```
@@ -22,10 +22,10 @@ Headless spiking neural-network runtime written in Rust.
 Requires **Rust 1.97.1 only** (`rust-toolchain.toml`). Do not use other toolchains.
 
 ```bash
-# Release build (includes soma-daemon)
-cargo build --release --bin soma-daemon
+# Release build (includes brainstem-daemon)
+cargo build --release --bin brainstem-daemon
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)